### ISSUE TITLE
Can't retrieve frames from timeline

### ISSUE THREAD

**Issue by RedBeren** [meraki-analytics/cassiopeia#97]
When going through the "Pulling Data and Data Types" example in the docs, I get

`AttributeError: 'NoneType' object has no attribute 'frames'`

when trying to access frames from the timeline. Here's the code I was using:

```{python}
from cassiopeia import riotapi

riotapi.set_region("NA")
riotapi.set_api_key("MY-KEY")

dyrus = riotapi.get_summoner_by_name("Dyrus")

match_list = dyrus.match_list()
matchreference = match_list[-1]
match = matchreference.match()

for frame in match.timeline.frames:
    print("The amount of gold for each participant at minute {} was:".format(int(frame.timestamp.seconds/60)))
    for participant, participant_frame in frame.participant_frames.items():
        print("  {summoner} had {amount} gold".format(summoner=participant_frame.participant.summoner.name, amount=participant_frame.gold))
```

I can get the match, participants, and timeline successfully, but not the frames.

**Comment by jjmaldonis** [comment]
Thanks for the report. The match taken from `match_list[-1]` is a match from 2014, and Riot doesn't keep timeline data back that far.

The docs should say `match_list[0]` instead, which gets the most recent game. I've updated the docs and I made sure your code works with that minor change.

Hope that helps and let us know if you run into other issues!

Feel free to close this issue if it's working for you too.
